data/mods/Aftershock/items/armor_helmets.json

```
@@ -0,0 +1,85 @@
+[
+  {
+    "id": "helmet_full_carbide",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "carbide raider's helmet" },
+    "description": "This helmet is designed for the individual that is worried about catching heavy weapons fire to the face.  It is unwieldy and heavy, and lacks basic environmental protection features, but it just might stop even the largest of ballistic rounds.",
+    "weight": "3175 g",
+    "volume": "2750 ml",
+    "price": 40000,
+    "price_postapoc": 285000,
+    "bashing": 10,
+    "material": [ "vacuum_carbide" ],
+    "symbol": "[",
+    "looks_like": "helmet_barbute",
+    "color": "light_gray",
+    "armor_portion_data": [
+      { "covers": [ "head" ], "coverage": 100, "encumbrance": 40 },
+      { "covers": [ "mouth" ], "coverage": 100, "encumbrance": 30 },
+      { "covers": [ "eyes" ], "coverage": 80, "encumbrance": 20 }
+    ],
+    "warmth": 10,
+    "material_thickness": 5,
+    "environmental_protection": 1,
+    "techniques": [ "WBLOCK_1" ],
+    "flags": [ "VARSIZE", "STURDY", "HELMET_COMPAT" ]
+  },
+  {
+    "id": "tac_fullhelmet_uicas",
+    "repairs_like": "xlkevlar",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "UICAS Tip of the Spear helmet" },
+    "description": "An all environment combat helmet issued to UICA espatiers.  Commonly known as "the jar"  due to its unusual, clear polymer canopy that leaves its wearers face visible.",
+    "weight": "1324 g",
+    "volume": "2 L",
+    "price": 50000,
+    "price_postapoc": 150000,
+    "to_hit": -1,
+    "bashing": 10,
+    "material": [ "nanoprinted_alloy" ],
+    "symbol": "[",
+    "looks_like": "helmet_motor",
+    "color": "dark_gray",
+    "armor_portion_data": [
+      { "covers": [ "head" ], "coverage": 100, "encumbrance": 20 },
+      { "covers": [ "eyes" ], "coverage": 100, "encumbrance": 5 },
+      { "covers": [ "mouth" ], "coverage": 100, "encumbrance": 8 }
+    ],
+    "warmth": 15,
+    "material_thickness": 5,
+    "environmental_protection": 2,
+    "techniques": [ "WBLOCK_1" ],
+    "flags": [ "VARSIZE", "STURDY", "SUN_GLASSES" ],
+    "environmental_protection_with_filter": 16,
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "gasfilter_m": 100 } } ],
+    "ammo": "gasfilter_m",
+    "use_action": [ "GASMASK" ]
+  },
+  {
+    "id": "tac_helmet_uicas",
+    "repairs_like": "xlkevlar",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "UICAS Point of Impact helmet" },
+    "description": "A lightweight black helmet that provides excellent protection from all sorts of damage, unless it's airborne.",
```

```suggestion
    "description": "A lightweight black helmet that provides excellent protection from all sorts of damage, unless it's airborne.  UICAS makes the point of designing all headgear whenever possible with clear faceplates, in order to provide a human touch to all interactions.",
```
What do you think of this addition @John-Candlebury?

---

I like it